src/validations.ts

```diff
@@ -1,4 +1,5 @@
 const emailRegex = /^([a-zñ\d[\]])(\.?([\wñ&/~\-+\][]+))*@([ñ\w.\-\][]+)\.[\w\-\][.]{2,}(\?)?.*$/i;
+const nameRegex = /^[\u00C0-\u1FFF\u2C00-\uD7FF\w][^#&<>[\](\)\"~;=+@:*$^%{}?]{1,}$/i;
```

Comment:
What do you think about the _negations_?

Maybe this alternative could work: `^[\u00C0-\u1FFF\u2C00-\uD7FF\w][\u00C0-\u1FFF\u2C00-\uD7FF\w'`\-\.\ ]+$`

Reply:
update